src/libexec/zmfixperms

```diff
@@ -698,4 +698,41 @@ fi
 if [ -d /opt/zimbra/data/rabbitmq ]; then
 	chown -R ${zimbra_user}:${zimbra_group} /opt/zimbra/data/rabbitmq
 fi
+
+
+##### Fix permissions for ldap and proxy #####
+
+if [ -x /opt/zimbra/common/sbin/nginx ]; then
+  chown ${zimbra_user}:${zimbra_group} /opt/zimbra/common/sbin/nginx
+  chmod 700 /opt/zimbra/common/sbin/nginx
+
+  # changing permission will reset capabilities so set it again
+  echo "Set capability for /opt/zimbra/common/sbin/nginx"
+  setcap CAP_NET_BIND_SERVICE=+ep /opt/zimbra/common/sbin/nginx
+fi
+
+if [ -x /opt/zimbra/bin/zmproxyctl ]; then
+  chown ${zimbra_user}:${zimbra_group} /opt/zimbra/bin/zmproxyctl
+  chmod 700 /opt/zimbra/bin/zmproxyctl
+fi
+
+if [ -x /opt/zimbra/libexec/zmslapd ]; then
+  chown ${zimbra_user}:${zimbra_group} /opt/zimbra/libexec/zmslapd
+  chmod 700 /opt/zimbra/libexec/zmslapd
+fi
+
+if [ -x /opt/zimbra/common/libexec/slapd ]; then
+  chown ${zimbra_user}:${zimbra_group} /opt/zimbra/common/libexec/slapd
+  chmod 700 /opt/zimbra/common/libexec/slapd
+
+  # changing permission will reset capabilities so set it again
+  echo "Set capability for /opt/zimbra/common/libexec/slapd"
+  setcap CAP_NET_BIND_SERVICE=+ep /opt/zimbra/common/libexec/slapd
+fi
+
+if [ -x /opt/zimbra/bin/ldap ]; then
+  chown ${zimbra_user}:${zimbra_group} /opt/zimbra/bin/ldap
+  chmod 700 /opt/zimbra/bin/ldap
+fi
+
```

Comment:
Why do you want the binaries to be owned by zimbra instead of root?
This again makes them vulnerable (writable) for attackers with local zimbra access.

Reply:
If you want to limit *execution* to the zimbra user, better use 750 root:zimbra instead.